experimenter/experimenter/jetstream/models.py

```diff
@@ -261,6 +261,12 @@ def append_conversion_count(self, primary_metrics_set):
                     primary_metric_data, BranchComparison.ABSOLUTE
                 )
 
+                if (
+                    absolute_primary_metric_vals.all is None
+                    or len(absolute_primary_metric_vals.all) == 0
+                ):
```

Comment:
`bool(None)` and `bool([])` both eval to `False`, so you can do `if not absolute_primary_metric_vals.all`